.gitlab/check.yml

```diff
@@ -0,0 +1,31 @@
+.check_skip_rust:
+  before_script:
+    - |-
+      if echo "$CI_COMMIT_TITLE" | grep -q "ci-skip-rust"; then
+        echo "Skipping rust tests due to commit title"
+        exit 0
+      fi
+
+.check_skip_integration_tests:
+  before_script:
+    - |-
+      if echo "$CI_COMMIT_TITLE" | grep -q "ci-skip-integration-tests"; then
+        echo "Skipping integration tests due to commit title"
+        exit 0
+      fi
+
+cargo-clippy:
+  extends: .check_skip_rust
+  image: paritytech/ci-unified:bullseye-1.77.0
+  stage: check
+  cache:
+    key: ${CI_COMMIT_REF_SLUG}-clippy
+    paths:
+    - target/
+    - $HOME/.cargo/bin/
+    - $HOME/.cargo/registry/index/
+    - $HOME/.cargo/registry/cache/
+    - $HOME/.cargo/git/db/
+  script:
+    - SKIP_WASM_BUILD=1 cargo clippy --all-targets --locked --workspace --quiet
```

Comment:
Is this necessary also when using `clippy`?

Reply:
copying best practices I guess